lib/commands/file-movement.js

```diff
@@ -47,12 +48,19 @@ async function parseContainerPath (remotePath, containerRootSupplier) {
     log.errorAndThrow(`It is expected that package identifier is separated from the relative path with a single slash. ` +
                       `'${remotePath}' is given instead`);
   }
-  const containerRoot = _.isFunction(containerRootSupplier) ?
-    (await containerRootSupplier(match[1])) :
-    containerRootSupplier;
+  let bundleId = match[1];
+  let containerType = null;
+  const typeSeparatorPos = bundleId.indexOf(CONTAINER_TYPE_SEPARATOR);
+  if (typeSeparatorPos > 0 && typeSeparatorPos < bundleId.length - 1) {
```

Comment:
I'd like it if there were JSDocs for this method since I'm not too sure what this is supposed to do.

It should have been added already, but better late than never

Reply:
done